src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class to represent tasks with deadlines.
+ */
+public class Deadline extends Task {
+    protected String deadline;
```

Comment:
Maybe the variable can be named as `description` instead to avoid confusion with the Deadline object created?